src/bitcoin-cli.cpp

```diff
@@ -867,6 +870,87 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+#ifndef WIN32
+    const std::string RESET = "\x1B[0m";
+    const std::string GREEN = "\x1B[32m";
+    const std::string BLUE = "\x1B[34m";
+    const std::string YELLOW = "\x1B[33m";
+    const std::string MAGENTA = "\x1B[35m";
+    const std::string CYAN = "\x1B[36m";
+#else
+    const std::string RESET = "";
+    const std::string GREEN = "";
+    const std::string BLUE = "";
+    const std::string YELLOW = "";
+    const std::string MAGENTA = "";
+    const std::string CYAN = "";
+#endif
+    const std::string BITCOIN_EMOJI = "\u20BF";
+
+    std::string result_string = strprintf("%sChain: %s%s\n", BLUE, result["chain"].getValStr(), RESET);
+    result_string += strprintf("Blocks: %s\n", result["blocks"].getValStr());
+    result_string += strprintf("Headers: %s\n", result["headers"].getValStr());
+    result_string += strprintf("Verification progress: %s\n", result["verificationprogress"].getValStr());
+    result_string += strprintf("Difficulty: %s\n\n", result["difficulty"].getValStr());
+
+    result_string += strprintf(
+        "%sNetwork: in %s, out %s, total %s%s\n",
+        GREEN,
+        result["connections"]["in"].getValStr(),
+        result["connections"]["out"].getValStr(),
+        result["connections"]["total"].getValStr(),
+        RESET);
+    result_string += strprintf("Version: %s\n", result["version"].getValStr());
+    result_string += strprintf("Time offset (s): %s\n", result["timeoffset"].getValStr());
+    const std::string proxy = result["proxy"].getValStr();
+    result_string += strprintf("Proxy: %s\n", proxy.empty() ? "N/A" : proxy);
+    result_string += strprintf("Relay fee (%s/kvB): %s\n\n", CURRENCY_UNIT, result["relayfee"].getValStr());
+
+    if (!result["has_wallet"].isNull()) {
+        const std::string walletname = result["walletname"].getValStr();
+        result_string += strprintf("%sWallet: %s%s\n", MAGENTA, walletname.empty() ? "\"\"" : walletname, RESET);
+
+        result_string += strprintf("Keypool size: %s\n", result["keypoolsize"].getValStr());
+        if (!result["unlocked_until"].isNull()) {
+            result_string += strprintf("Unlocked until: %s\n", result["unlocked_until"].getValStr());
+        }
+        result_string += strprintf("Pay transaction fee (%s/kvB): %s\n\n", CURRENCY_UNIT, result["paytxfee"].getValStr());
```

Comment:
So that it is clear that this line refers to the `-paytxfee` configuration option without having to look in the code, and that it is actually a fee rate, maybe either s/Pay transaction fee/Paytxfee/ or
```suggestion
        result_string += strprintf("Transaction fee rate (-paytxfee) (%s/kvB): %s\n\n", CURRENCY_UNIT, result["paytxfee"].getValStr());
```
See also src/wallet/init.cpp:
```cpp
argsman.AddArg("-paytxfee=<amt>", strprintf("Fee rate (in %s/kvB) to add to transactions you send (default: %s)",
```

Reply:
Updated to `Paytxfee` instead because `Transaction fee rate (-paytxfee)` seems a bit too long